Components/ListIcons/README.md

```diff
@@ -0,0 +1,13 @@
+Title: List Icons
+
+---
+
+Category: list
```

Comment:
The word `List` in the `Category` should be capitalized to make it consistent a cross flynt.